src/databricks/labs/ucx/aws/access.py

```diff
@@ -44,7 +42,6 @@ def __init__(
         self._backend = backend
         self._ws = ws
         self._locations = external_locations
-        self._schema = schema
         self._aws_account_id = aws_account_id
         self._kms_key = kms_key
         self._filename = self.INSTANCE_PROFILES_FILE_NAMES
```

Comment:
this one is not needed - this is the copy of the constant on the same class

Reply:
removed